templates/Wettbewerbe/ein_wettbewerb.html

```diff
@@ -21,4 +21,40 @@ <h2>Wettbewerb {{ wettbewerb.name }} ({{ wettbewerb.art }})</h2>
       <li><a style="color: gray;" href="mich_eintragen/">[meinen Erfolg eintragen]</a></li>
     {% endif %}
   </ul>
-{% endblock %}
+
+  {# Links zur Veranstaltung und anderen Wettbewerben der Veranstaltung #}
+  {% if wettbewerb.veranstaltung %}
+  Der Wettbewerb hat im Rahmen der Veranstaltung <a href={{wettbewerb.veranstaltung.get_absolute_url}}>{{ wettbewerb.veranstaltung.name }}</a> stattgefunden
+    {% ifnotequal wettbewerb.veranstaltung.count 1 %}
+    Weitere Wettbewerbe auf dieser Veranstaltung:
+    <ul>
+      {% for w in wettbewerb.veranstaltung.wettbewerbe.all %}
+        {% ifnotequal w wettbewerb %}
+          <li><a href={{w.get_absolute_url}}>{{ w.name }}</a></li>
+        {% endifnotequal %}
+      {% endfor %}
+    </ul>
+    {% endifnotequal %}
+  {% endif %}
+
+  {# Link zur nächsten/vorherigen Runde #}
+  {% if fortsetzung %}
+    zur nächsten Runde: <a href={{fortsetzung.get_absolute_url}}>{{fortsetzung.name}}</a>
+  {% endif %}
+  {% if vorherige %}
+    zu Wettbewerben der vorherigen Runde:
+    {% for w in vorherige %}
+      <ul>
+      <li><a href={{w.get_absolute_url}}>{{w.name}}</a></li>
+      </ul>
+    {% endfor %}
+  {% endif %}
+  {% if nachbarn %}
+    gleichzeitig finden andere Wettbewerbe zu dieser Wettbewerbsrunde statt:
+    <ul>
+    {% for w in nachbarn %}
+      <li><a href={{w.get_absolute_url}}>{{w.name}}</a></li>
+    {% endfor %}
+    </ul>
+  {% endif %}
+{% endblock %}
```

Comment:
Mach mal bitte überall Leerzeichen rein, also {{ zeug }} statt {{zeug}}

Reply:
ich finds persönlich schöner ohne die Leerzeichen, weil sublime das ganze dann farblich hervorhebt aber ich änder es auch gerne

Reply:
ja, da kann man sich streiten, aber die offiziellen Beispiele sind überall mit Leerzeichen. 
Und manchmal habei in templates Leerzeichen sogar Bedeutung, das hattest du ja sogar erfahren: {% if zeug.count>1 %} ging nicht, es muss {% if zeug.count > 1 %} sein.